umap/static/umap/js/modules/utils.js

```diff
@@ -22,3 +22,20 @@ export function checkId(string) {
   if (typeof string !== 'string') return false
   return /^[A-Za-z0-9]{5}$/.test(string)
 }
+
+export function getImpactsFromSchema(fields) {
```

Comment:
I'd make a few unitests (JS) for this one :)

Reply:
✔️